docker/simplified_app.sh

```diff
@@ -46,7 +46,8 @@ cd circulation
 git checkout $version
 
 # Use https to access submodules.
-git config submodule.core.url https://github.com/NYPL-Simplified/server_core.git
+git submodule init
```

Comment:
This has to be called in order to initialize the config that is changed below. It initializes the config, but doesn't actually checkout the module. From the [git manual](https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt-init--ltpathgt82308203).

> Initialize the submodules recorded in the index (which were added and committed elsewhere) by setting submodule.$name.url in .git/config. ...
> You can then customize the submodule clone URLs in .git/config for your local setup and proceed to git submodule update; you can also just use git submodule update --init without the explicit init step if you do not intend to customize any submodule locations.